app/lib/pdf.js

```diff
@@ -64,6 +64,34 @@ async function get(url, options = {}) {
                 throw e;
             });
 
+        /*
+         * This works around an issue with puppeteer not printing canvas
+         * images that were loaded from a file.
+         * It is likely this issue: https://bugs.chromium.org/p/chromium/issues/detail?id=809065
+         * (though not WebGL-related as some of the commenters suggest)
+         */
+        await page.evaluate(() => {
+            /* eslint-env browser */
```

Comment:
It's unclear what the scope of this eslint-env override is. Setting it to `eslint-env: node` after this page.evaluate function doesn't actually seem to switch it back.